TextToSpeech/impl/TTSSpeaker.cpp

```diff
@@ -129,6 +148,9 @@ TTSSpeaker::TTSSpeaker(TTSConfiguration &config) :
     m_pipeline(NULL),
     m_source(NULL),
     m_audioSink(NULL),
+#if defined(PLATFORM_REALTEK)
```

Comment:
instead of guarding m_audioVolume in all places, assign m_audioVolume as m_audioSink if the platform is not realtek, this way we can avoid few compiler flag checks

Reply:
The AMLOGIC code was already there and not made by me

Reply:
We cannot do this because m_audioSink is used to create the actual sink element (soc-specific sink)